specs/waku/v2/waku-relay-security-analysis.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Security Analysis of Waku Relay
+version: 
+status: Draft
+authors: Sanaz Taheri <sanaz@status.im>
+---
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+  - [Terminologies](#terminologies)
+  - [Anonymity](#anonymity)
+    - [Definition](#definition)
+      - [Message Publisher Anonymity](#message-publisher-anonymity)
+      - [Topic Subscriber Anonymity](#topic-subscriber-anonymity)
+    - [Analysis](#analysis)
+  - [Authenticity](#authenticity)
+    - [Definition](#definition-1)
+    - [Analysis](#analysis-1)
+  - [Integrity](#integrity)
+    - [Definition](#definition-2)
+    - [Analysis](#analysis-2)
+  - [Confidentiality](#confidentiality)
+    - [Definition](#definition-3)
+    - [Analysis](#analysis-3)
+  - [Changelog](#changelog)
+    - [2.0.0-beta1](#200-beta1)
+- [Copyright](#copyright)
+- [References](#references)
+
+# Abstract
+
+In this spec, we analyze the security of the  `relay` protocol concerning data confidentiality, integrity, authenticity, and anonymity. This is to enable users of this protocol to make informed decision about all the secuirty properties that can or can not achieve by deploying `relay` protocol. 
+
+## Terminologies
+
+Honest but Curious / static adversary: Throught this spec we consider an Honest but Curious adversarial model (static adversary interchangebly. That is, an adversarial entity may attempt to collect infromation from other peers (i.e., being curious) in order to succedd in its attack but it does so without violating protocol definitions and instructions (is honest), namely, it does follow the protocol specifications. 
+
+Personally identifiable information (PII): PII indicates any piece of data that can be used to uniquely identify a Peer. For example, the signature verification key, and the hash of one's IP address are unique for each peer and hence count as PII.
+
+## Anonymity
+### Definition
```

Comment:
Threat model and definition seems like good sections in the main relay spec

Reply:
> Threat model and definition seems like good sections in the main relay spec

I embedded the security analysis into the relay specs